src/main.rs

```diff
@@ -9,18 +9,28 @@
 extern crate diesel;
 #[macro_use]
 extern crate rocket;
+#[macro_use]
+extern crate tracing;
 
 mod application;
 mod errors;
 mod routes;
 mod storage;
 mod web;
 
+use tracing_subscriber::{fmt, EnvFilter};
+
 fn main() {
+    // set up logging of applications events to stdout
+    let subscriber = fmt()
+        .with_env_filter(EnvFilter::from_default_env())
```

Comment:
Is this the line that configures tracing to pull format from `RUST_LOG`? Shouldn't we maybe use a single source for configuration? In our case Rocket for now?

Reply:
Do you mean just some environment variable prefixed with `ROCKET_`? Or `ROCKET_LOG` specifically? The latter may be problematic, because Rocket's logging levels do _not_ map directly to the levels used in `tracing` :( TBH, I'd rather disable Rocket's logging completely other than printing the configuration on start.